docs/_docs/templates.md

```diff
@@ -421,56 +421,164 @@ The default is `default`. They are as follows (with what they filter):
-if `_posts/2014-09-03-my-file.markdown` uses the `include_relative` tag, the included file
-must be within the `_posts` directory, or one of its subdirectories. You cannot include
-files in other locations.
+This include contains 5 parameters:
+    
+* `url`
+* `max-width`
+* `file`
+* `alt`
+* `caption`
 
-All the other capabilities of the `include` tag are available to the `include_relative` tag,
-such as using variables.
+To account for optional parameters, the include uses `if` tags with the parameters. For example, `{% raw %}if {{include.url}}{% endraw %}` will include the `url` only if the `url` parameter is specified in the include.
+
+Here's an example that passes all the parameters to this include (the include file is named image.html):
+
+```liquid
+{% raw %}{% include image.html url="http://jekyllrb.com" 
+max-width="200px" file="logo.png" alt="Jekyll logo" 
+caption="This is the Jekyll logo." %} {% endraw %}
+```
+
+The result is the original HTML code shown earlier. 
+
+You can create includes that act as templates for a variety of uses &mdash; inserting audio or video clips, alerts, special formatting, and more.
+
+### Passing parameter variables to includes
+
+Suppose the parameter you want to pass to the include is a variable rather than a string. For example, you might be using {% raw %}`{{site.product_name}}`{% endraw %} to refer to every instance of your product rather than the actual hard-coded name. (In this case, your _config.yml file would have a key called `product_name` with a value of your product's name.)
+
+The string you pass to your include parameter can't contain curly braces. For example, you can't pass a parameter that contains this: {% raw %}`"The latest version of {{site.product_name}} is now available."`{% endraw %} 
+
+If you want to include this variable in your parameter that you pass to an include, you need to store the entire parameter as a variable before passing it to the include. You can use `capture` tags to create the variable:
+
+```liquid
+{% raw %}{% capture download_note %}The latest version of 
+{{site.product_name}} is now available.{% endcapture %}{% endraw %}
+```
+
+Then pass this captured variable into the parameter for the include. Omit the quotation marks around the parameter content because it's no longer a string (it's a variable):
+
+```liquid
+{% raw %}{% include note.html content=download_note %}{% endraw %}
+```
+
+### Passing references to YAML files as parameter values
+ 
+Instead of passing string variables to the include, you can pass a reference to a YAML data file stored in the \_data folder. 
+
+Here's an example. In the \_data folder, suppose you have a YAML file called profiles.yml. Its content looks like this:
+ 
+```yaml
+- name: John Doe
+  login_age: old
+  image: johndoe.jpg
+
+- name: Jane Doe
+  login_age: new
+  image: janedoe.jpg
+```
+
+In the \_includes folder, you have a file called spotlight.html with this code:
```

Comment:
It's a directory, not a folder.

The slash goes the other way, but is not needed, since `_includes` is at the root of the site source.

`_includes` should be enclosed in backticks.